src/Neo.VM/Utility.cs

```diff
@@ -27,6 +27,23 @@ static Utility()
             StrictUTF8.EncoderFallback = EncoderFallback.ExceptionFallback;
         }
 
+        public static bool NotZero(this ReadOnlySpan<byte> x)
+        {
+            int i;
+            for (i = 0; i + 7 < x.Length; i += 8)
+            {
+                if ((x[i] | x[i + 1] | x[i + 2] | x[i + 3] | x[i + 4] | x[i + 5] | x[i + 6] | x[i + 7]) != 0)
```

Comment:
This won't be faster as previous version

Reply:
> This won't be faster as previous version



> This won't be faster as previous version

The performance has been optimized,